frontend/src/scenes/events/EventElements.tsx

```diff
@@ -10,17 +11,19 @@ export function EventElements({ event }: { event: EventType }): JSX.Element {
     elements = elements.slice(Math.max(elements.length - 10, 1))
 
     return (
-        <div>
+        <div
+            className="p-4 m-2 rounded"
+            style={{
+                backgroundColor: 'rgb(39, 40, 34)', // consistent with JSON bg okaidia color
```

Comment:
This is an "okapi"

![image](https://user-images.githubusercontent.com/53387/188600604-91570a29-346a-42af-ab13-a7e80ec4c0c1.png)

But what's an "okaidia"?

I saw lower down it has to do with syntax highlighting. Might be worth referring to the component (e.g. `okaida syntax highlighter color`)